commands/alias.js

```diff
@@ -37,7 +41,7 @@ module.exports = class Version {
             const messages = await sendCommand({
                 host: this.server,
                 method: 'PUT',
-                pathname: `/${this.org}/pkg/${this.name}/v${this.alias}`,
+                pathname: `/${this.org}/${this.type}/${this.name}/v${this.alias}`,
```

Comment:
If you do `path.join(this.org, this.type, this.name, this.alias)` you do not have to deal with adding `/` and stuff.

Reply:
good point, not sure why I forgot that